src/sentry/search/eap/columns.py

```diff
@@ -80,6 +64,20 @@ def meta_type(self) -> str:
             return self.search_type
 
 
+@dataclass(frozen=True)
+class ResolvedColumn(ResolvedAttribute):
+    # The internal rpc alias for this column
+    internal_name: str = ""
```

Comment:
Hmmm good point, my original thinking was that if its not set "" is better than error, but found `kw_only` for dataclass, because otherwise ResolvedColumn gets mad that `internal_name` follows default args